install_files/appstream/io.github.openrazer.openrazer.metainfo.xml

```diff
@@ -74,6 +74,7 @@
     <modalias>usb:v1532p0232d*</modalias>
     <modalias>usb:v1532p0233d*</modalias>
     <modalias>usb:v1532p023bd*</modalias>
+    <modalias>usb:v1532p023ad*</modalias>
```

Comment:
You can use a script in `scripts/` in the repo to generate this file, or , move the line manually one up